qemu/tests/virtio_trace_pipenb.py

```diff
@@ -0,0 +1,29 @@
+from avocado.utils import process
+from virttest import error_context
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Under named-pipe non-blocking testing:
+    1) Create pipe named by the following
+    2) Boot up a single-CPU guest with a virtio-serial device and
+       named-pipe chardev backend
+    3) Run a background work on the guest.
+    4) Write data to the virtio-serial port until the guest stops.
+    5) Read the named-pipe file on the host.
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+    timeout = float(params.get("login_timeout", 240))
+    vm = env.get_vm(params["main_vm"])
+    serials = params["serials"].split()
+    v_path = vm.get_serial_console_filename(serials[-1])
+    vm.verify_alive()
+    session = vm.wait_for_login(timeout=timeout)
+    session.cmd_status_output(" while :;do sleep 1; date; done &")
+    session.cmd_status_output("cat /proc/kallsyms > /dev/virtio-ports/vs2 &")
+    out_put = process.getoutput("cat {}".format(v_path), timeout=60, shell=True)
+    assert 'compile_date_show' in out_put
```

Comment:
@nanliu-r  Please help confirm the check point of this case, thanks!

Reply:
@nanliu-r  please check, review from product perspective first, thanks!

Reply:
sorry for the late reply, the executed result is expected.

Reply:
> @nanliu-r Please help confirm the checkpoint of this case, thanks!

@michiboo Can Micky explain why check this parameter?

Reply:
It just show there is output for this command